pulsar-client-cpp/lib/UnAckedMessageTrackerEnabled.h

```diff
@@ -25,25 +25,27 @@
 namespace pulsar {
 class UnAckedMessageTrackerEnabled : public UnAckedMessageTrackerInterface {
    public:
-    ~UnAckedMessageTrackerEnabled();
+       virtual ~UnAckedMessageTrackerEnabled() { this->close(); }
```

Comment:
It won't pass the clang-format check here. Please use clang-format 5.0 to reformat your code.